Modules/timemodule.c

```diff
@@ -1005,18 +1005,23 @@ time_mktime(PyObject *self, PyObject *tup)
     {
         return NULL;
     }
-#ifndef _AIX
-    buf.tm_wday = -1;  /* sentinel; original value ignored */
-    tt = mktime(&buf);
-#else
+
+#if defined(_AIX) || (defined(__VXWORKS__) && !defined(_WRS_CONFIG_LP64))
```

Comment:
VxWorks defines time_t as long. In 32-bit mode, time_t can contain only 4 bytes. So definitely 2038 issue exists on VxWorks. We will report OverflowError for this. However, in 64-bit  mode, long represents 8 bytes. There will be no issue.
`typedef long time_t;`